package.json

```diff
@@ -27,32 +28,35 @@
   "homepage": "https://github.com/silvermine/videojs-chromecast#readme",
   "dependencies": {
     "class.extend": "0.9.1",
-    "underscore": "1.8.3",
```

Comment:
Looks like this was messed up before. `underscore` was listed in the `devDependencies` and the `dependencies`. We do use it though in the code, so it should be in `dependencies`.